### PR TITLE
shelter partial functional

### PR DESCRIPTION
**Pull request by perryr16**
shelter create and update partial

### CODE REVIEW

File: app/assets/stylesheets/application.css [hunk 1]
```diff
@@ -47,7 +47,10 @@
 
    margin-top: 80px;
    padding: 10px;
-   border-left: solid 8px #0077A6;
+   /* border-left: solid 8px #0077A6; */
+   border-right: solid 8px yellow;
+   border-bottom: solid 8px yellow;
```
Comment (perryr16) [inline app/assets/stylesheets/application.css:52]: black and yellow black and yellow

Comment (muydanny) [reply app/assets/stylesheets/application.css:52]: wizzzz

File: app/views/shelters/_shelter_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<%= form_tag path, method: method do %>
+  <%= label_tag :name%><br>
```
Comment (perryr16) [inline app/views/shelters/_shelter_form.html.erb:2]: our first partial!

Comment (muydanny) [reply app/views/shelters/_shelter_form.html.erb:2]: outstanding

File: app/views/shelters/_shelter_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  <%= text_field_tag :name, name_fillin%><br><br>
+  <%= label_tag :address%><br>
+  <%= text_field_tag :address, address_fillin%><br><br>
```
Comment (perryr16) [inline app/views/shelters/_shelter_form.html.erb:5]: fillin represents an autopopulate value

**Pull request merged**